_studio/mfx_lib/encode_hw/h265/src/mfx_h265_encode_vaapi.cpp

```diff
@@ -1053,6 +1091,10 @@ mfxStatus VAAPIEncoder::CreateAccelerationService(MfxVideoParam const & par)
 
     MFX_CHECK_WITH_ASSERT(MFX_ERR_NONE == SetFrameRate(par, m_vaDisplay, m_vaContextEncode, VABufferNew(VABID_FrameRate, 1)), MFX_ERR_DEVICE_FAILED);
     MFX_CHECK_WITH_ASSERT(MFX_ERR_NONE == SetQualityLevelParams(par, m_vaDisplay, m_vaContextEncode, VABufferNew(VABID_QualityLevel, 1)), MFX_ERR_DEVICE_FAILED);
+    if(par.m_ext.CO2.MaxSliceSize != 0)
+    {
+        MFX_CHECK_WITH_ASSERT(MFX_ERR_NONE == SetMaxSliceSize(par, m_vaDisplay, m_vaContextEncode, VABufferNew(VABID_MaxSliceSize, 1)), MFX_ERR_DEVICE_FAILED);
```

Comment:
I suggest to pull this function out of macro. In debug build expansion of this macro leads to double call of function. It would be nice if you will fix this issue for other similar call which I see in near by code